lib/instances/GuiService.lua

```diff
@@ -1,4 +1,33 @@
 local BaseInstance = import("./BaseInstance")
 local GuiService = BaseInstance:extend("GuiService")
 
+local function validateType(name, value, expectedTypeAsString)
+	local actualType = type(value)
```

Comment:
Can we use `typeof` from Lemur's function set instead of `type` here? We can get slightly better error messages that way.